drivers/bluetooth/hci/Kconfig.infineon

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if BT_CYW43XXX
+
+choice  CYW43XXX_PART
+	prompt "Select CYW43XXX part"
+
+config CYW4343W
+	bool "CYW4343W"
+	help
+	  Enable Infineon CYW4343W BLE connectivity,
+	  More information about CYW4343W device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw4343w/
+
+config CYW4373
+	bool "CYW4373"
+	help
+	  Enable Infineon CYW4373 BLE connectivity,
+	  More information about CYW4373 device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw4373/
+
+config CYW43012
+	bool "CYW43012"
+	help
+	  Enable Infineon CYW43012 BLE connectivity,
+	  More information about CYW43012 device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw43012/
+
+config CYW43438
+	bool "CYW43438"
+	help
+	  Enable Infineon CYW43438 BLE connectivity,
+	  More information about CYW43438 device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw43438/
+
+config CYW43439
+	bool "CYW43439"
+	help
+	  Enable Infineon CYW43439 BLE connectivity,
+	  More information about CYW43439 device you can find on
+	  https://www.infineon.com/cms/en/product/wireless-connectivity/airoc-wi-fi-plus-bluetooth-combos/cyw43439/
+
+config BT_CYW43XXX_CUSTOM
+	bool "Custom CYW43xx device/module"
+	help
+	  Select Custom CYW43xx device/module. For this option,
+	  user must to provide path to BT firmware HCD file for
+	  custom or vendor CYW43xx modules in CYW43XX_CUSTOM_FIRMWARE_HCD_BLOB.
+
+endchoice
+
+choice CYW43012_MODULE
+	prompt "Select CYW43012 module"
+	depends on CYW43012
+
+config CYW43012_MURATA_1LV
+	bool "MURATA-1LV"
+	help
+	  Type 1LV is a small and high performance module based on
+	  Infineon CYW43012 combo chipset which supports Wi-Fi® 802.11a/b/g/n +
+	  Bluetooth® 5.0 BR/EDR/LE up to 72.2Mbps PHY data rate on Wi-fi® and
+	  3Mbps PHY data rate on Bluetooth®. 2Mbps LE PHY is also supported.
+
+	  The CYW43012 implements highly sophisticated enhanced collaborative
+	  coexistence hardware mechanisms and algorighms, which ensure that WLAN
```

Comment:
typo: algorithms.

Also this kinda reads like an ad :smile: is all of the text necessary?
Ie this sentence could just be
```
The CYW43012 implements radio coexistence, optimizing for maximum WLAN and Bluetooth performance. 
```
I'm not saying we can't have it, just saying maybe it belongs in an .rst in the documentation of the module/board/soc.